### PR TITLE
Add information from onboarding induction pack

### PR DESCRIPTION
**Pull request by Kellym-Kainos**

### CODE REVIEW

File: docs/onboarding/index.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+All private beta users will begin in the SANDBOX ENVIRONMENT. You may dual-key and/or to test with dummy data.
+{: .govuk-body}
+
+When you are ready, you can request access to the PRE-PRODUCTION and PRODUCTION ENVIRONMENT. Instead of dummy email addresses like those that were provided for SANDBOX, we will be using your users’ email addresses as the login username. Please avoid using dummy data within PRE-PRODUCTION and treat it as if it was the live system.
+{: .govuk-body}
+
+Two weeks’ notice will be required when you would like to move onto PRE-PRODUCTION/PRODUCTION. We will also need to ensure that you are paired up with a works promoter and local highway authority (LHA) within your geographical area accordingly.
+{: .govuk-body}
+
+Important: If you dual-key in EToN and the PRODUCTION, there will be duplicate records in the live system and must be avoided.
```
Comment (alistaircowan) [inline docs/onboarding/index.md]: Is this correct?

File: docs/onboarding/index.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+This contact should be the primary person that we can contact about your organisation’s use of Street Manager and has sufficient authority to make decisions around when to transition from the SANDBOX environment to PRE-PRODUCTION/PRODUCTION, and who the users will be taking part in Private Beta.
+{: .govuk-body}
+
+Responsibilities include but are not limited to the following:
+{: .govuk-body}
+
+<ol class="govuk-list govuk-list--bullet">
+  <li>Ensure that we are provided with details for your organisation for onboarding and testing/transition approach.</li>
+  <li>Agree with Street Manager’s terms of use. These will be sent to you when they are ready, targetted for the end of May.</li>
+  <li>[Agree to charging model for Street Manager and provide details for invoicing/payment (details TBC).]</li>
```
Comment (alistaircowan) [inline docs/onboarding/index.md]: Should this be in here?

File: docs/onboarding/index.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+  </ul>
+  <li>Take part in usability testing sessions – our user researchers may want to get your feedback on upcoming features</li>
+  <li>Communicate with your Private Beta Manager – If you need help or need to leave the test, please let us know. Keep us in the loop if your circumstances change at any point.</li>
+  <li>Store your login details securely.</li>
+  <li>Do not share information provided as part of private beta with anyone who is not participating. Information will be shared with you, some of which may be in draft to ask for your views. The project will make information more widely available as and when it has been agreed by the project team.</li>
+</ol>
+
+<hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+
+## Test scenarios
```
Comment (alistaircowan) [inline docs/onboarding/index.md]: Do we need a point in this subsection (or Responsibilities section above) around the fact all orgs should be coming up with their own functionality test plans/scenarios?

**Comment by alistaircowan** [comment]
Also some comments for @stevenalexander review

File: docs/onboarding/index.md (hunk 1 above)
Comment (esmemo) [reply docs/onboarding/index.md]: The notice period is ok, but for API, I don't think we need to pair them up as they get their own promoter and HA accounts

File: docs/onboarding/index.md (hunk 2 above)
Comment (esmemo) [reply docs/onboarding/index.md]: I thought Paul removed the reference to charging model. Let's just delete that.

Comment (esmemo) [reply docs/onboarding/index.md]: Which version of the document did you copy into github?

File: docs/onboarding/index.md (hunk 3 above)
Comment (esmemo) [reply docs/onboarding/index.md]: I was speaking with Steven about this and we were thinking to create some test scenarios for the API users at some point.

In the mean time, we should delete that section all together from here.

**Comment by esmemo** [comment]
Esme's comments overall

- Delete "Progress so far" as that's mostly just about the UI side
- The release aspect is specific to the UI side, how do we want to advise of new API parts being added? 
- Known issues and workarounds, do we want special comments in this area about the API as I imagine that some of it may not apply to them and some of it might. Not sure until we look at them. I'd advise deleting that section if we think most of it won't apply to them.
- delete the link to the API site?

**Comment by alistaircowan** [comment]
Thanks @esmemo I'll cleanup the PR and re-submit shortly......

File: docs/onboarding/index.md (hunk 1 above)
Comment (alistaircowan) [reply docs/onboarding/index.md]: Updated based on conversation with Steven.

File: docs/onboarding/index.md (hunk 2 above)
Comment (alistaircowan) [reply docs/onboarding/index.md]: Removed.

**Comment by alistaircowan** [comment]
This onboarding document is intended for all Private Beta users - frontend and API.  Therefore it should remain at a general-service level, though certain sections can cover specific elements as long as they call that out.  Therefore Progress so far can stay but I've modified to call out the UI focus.  Regarding release approach, I think we should use the same comms approach for both API and UI.

**Pull request merged**